airbyte-integrations/connectors/source-facebook-marketing/unit_tests/integration/config.py

```diff
@@ -25,7 +25,10 @@ class ConfigBuilder:
     def __init__(self) -> None:
         self._config: MutableMapping[str, Any] = {
             "account_ids": [ACCOUNT_ID],
-            "access_token": ACCESS_TOKEN,
+            "credentials": {
```

Comment:
There is no need to replace the tests, but `extend` them with the new functionality, using parametrized test case. With this changes, we loose the backward compatibility tests for the existing Customers.